### PR TITLE
created sample data for seeding database, and added necessary tests

### PR DESCRIPTION
**Pull request by matgbx**
here is the ticket:

https://trello.com/c/1kNrTJG8/3-generate-seed-dummy-data

### CODE REVIEW

File: src/spec/spec.js [hunk 1]
```diff
@@ -1,59 +1,110 @@
...
-    expect(Array.isArray(entreeMenu)).toBe(true);
-  });
+    it('should have a length of 20', () => {
+      expect(entreeMenu.length).toEqual(20);
+    });
 
-  it('should have a length of 20', () => {
-    expect(entreeMenu.length).toEqual(20);
-  });
+    it('should contain objects with properties: foodItem, cost, tags', () => {
```
Comment (tmkuba) [inline src/spec/spec.js:20]: expect.objectContaining(object) might be useful for here and the below test

File: src/spec/spec.js [hunk 2]
```diff
@@ -1,59 +1,110 @@
...
-    expect(typeof entreeMenu[0].foodItem).toBe('string');
-    expect(typeof entreeMenu[0].cost).toBe('number');
-    expect(typeof entreeMenu[0].tags).toBe('string');
+  describe('dessert generator func', () => {
+    it('should be an array', () => {
+      expect(Array.isArray(dessertMenu)).toBe(true);
+    });
+
+    it('should have a length of 8', () => {
+      expect(dessertMenu.length).toEqual(8);
```
Comment (tmkuba) [inline src/spec/spec.js:40]: nitpick, but there is a .hasLength() in jest, but not really a big deal

**Review by tmkuba — APPROVED** [review]
small suggestions for the expect() assertions.  not blocking though

**Review by leungnamson1023 — COMMENTED** [review]
test looks good

**Pull request merged**